### DISCUSSION TITLE
Auto disable user if user dont exist keycloak

### DISCUSSION THREAD

**Discussion by patsevanton** [netbirdio/netbird#1257] [Issue Triage]
Hi, Team.
I am using netbird with Keycloak OIDC client.
I dont want public Keycloak OIDC.
May be netbird auto disable user if user dont exist keycloak ?

**Comment by mlsmaycon** [comment]
Hello @patsevanton, you want the user to be disabled as part of the integration? On the latest version you can remove the user from the  dashboard when deleting it.

**Reply by patsevanton** [reply]
I can remove the user from dashboard by hand.
But may netbird autoremove or disable user if user dont exist keycloak/Active Directory ?
